components/i2c/bme280/definition.json

```diff
@@ -0,0 +1,28 @@
+{
+    "displayName":"BME280",
+    "i2cAddresses":[
+       "0x76, 0x77"
```

Comment:
This is wrong, should be `"i2cAddresses": [ "0x76", "0x77" ],`

Reply:
Resolved in https://github.com/adafruit/Wippersnapper_Components/pull/12/commits/d0b76a5364adf4c8f9fc9b32345b744424971d48